data/json/vehicleparts/battery.json

```diff
@@ -50,6 +50,21 @@
       { "item": "scrap", "count": [ 4, 7 ] }
     ]
   },
+  {
+    "id": "large_storage_battery",
+    "copy-from": "storage_battery",
+    "type": "vehicle_part",
+    "name": "large storage battery",
+    "item": "large_storage_battery",
+    "difficulty": 3,
```

Comment:
You *must* remove the `difficulty` line and instead add an appropriate `requirements` line as per the other storage batteries.

As you currently have it defined, this part will be repairable once installed, which is not intended to be permitted.

Reply:
That's exactly how it's done for other storage batteries (they're using copy-from). Should I fix them too while at it?

Reply:
if you look at the base "storage battery" object, it has no difficulty and no copy-from. That's the one I spent the most time testing.

I'd overlooked that the small and medium storage batteries have a difficulty, so perhaps it isn't as critical to remove it as I thought, but I expect as you have it currently configured you'll be able to repair a damaged large storage battery installed in a vehicle.

Admittedly, I expect this primarily because I keep having people with ask why their large storage batteries can be repaired but the others can't. If your testing demonstrates otherwise, I'm prepared to be corrected.